Documentation/ApiOverview/PageTypes/Index.rst

```diff
@@ -22,22 +22,22 @@ allowed on a certain page type.
 This is the default array as set in :file:`EXT:core/ext_tables.php`::
 
    $GLOBALS['PAGES_TYPES'] = [
-      (string)\TYPO3\CMS\Core\Domain\Repository\PageRepository::DOKTYPE_BE_USER_SECTION => [
```

Comment:
This is actually correct, at least for TYPO3v10+

See https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Deprecation-88746-PageRepositoryPHPClassMovedFromFrontendToCoreExtension.html

If you switch to the v9 docs, you can see the old class:

https://docs.typo3.org/m/typo3/reference-coreapi/9.5/en-us/ApiOverview/PageTypes/Index.html